### PR TITLE
Allow travis-ci failures on arm64

### PR DESCRIPTION
**Pull request by ChinouneMehdi**
& Build pFUnit with Debug-mode to get more information on failures.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -81,15 +87,15 @@ before_script:
    # Now build pFUnit
    - cd ${TRAVIS_BUILD_DIR}
    - mkdir -p build && cd build
-   - cmake .. -DCMAKE_Fortran_COMPILER=${FC} -DCMAKE_INSTALL_PREFIX=${HOME}/Software/pFUnit
+   - cmake .. -DCMAKE_Fortran_COMPILER=${FC} -DCMAKE_BUILD_TYPE=Debug -DCMAKE_INSTALL_PREFIX=${HOME}/Software/pFUnit
```
Comment (tclune) [inline .travis.yml:90]: Note pFUnit uses debug flags regardless of CMAKE_BUILD_TYPE.    I could not see any good reason to ever build without them for this utility.   Even my most expensive project with something like 2500 tests runs them all in a couple of seconds.

File: .travis.yml [hunk 2]
```diff
@@ -81,15 +87,15 @@ before_script:
...
 
 script:
    # Build
    - make -j$(nproc) VERBOSE=1
    # Make tests
    - make -j$(nproc) tests
-   # Run tests. If the first fails, run with -VV to try and diagnose errors
-   - ctest -j $(nproc) || ctest -VV
+   # Run tests
+   - ctest -j $(nproc) --output-on-failure
```
Comment (tclune) [inline .travis.yml:98]: Nice.  Did not know about that option.   @mathomp4 please take note for our other projects

File: .travis.yml [hunk 3]
```diff
@@ -20,6 +20,7 @@ addons:
       packages:
          - gfortran-8
          - gfortran-9
+         - libgfortran5-dbg
```
Comment (tclune) [inline .travis.yml:23]: What does this line do?

File: .travis.yml (hunk 2 above)
Comment (mathomp4) [reply .travis.yml:98]: Neither did I. I like it.

File: .travis.yml (hunk 3 above)
Comment (ChinouneMehdi) [reply .travis.yml:23]: this package contains debug symbols of libgfortran, which I think would help to get clear error messages.

File: .travis.yml (hunk 1 above)
Comment (ChinouneMehdi) [reply .travis.yml:90]: I will remove it.

Comment (ChinouneMehdi) [reply .travis.yml]: Done

**Pull request merged**